frappe/patches/v14_0/add_manage_subscriptions_in_navbar_settings.py

```diff
@@ -0,0 +1,25 @@
+import frappe
+
+
+def execute():
+	navbar_settings = frappe.get_single("Navbar Settings")
+
+	if frappe.db.exists("Navbar Item", {"item_label": "Manage Subscriptions"}):
+		return
+
+	for navbar_item in navbar_settings.settings_dropdown[5:]:
+		navbar_item.idx = navbar_item.idx + 1
```

Comment:
Keep patches simple. Just reset the idx after inserting a row in the middle instead of doing index arithematic :smile: 

```python
for idx, row in enumerate(child_table, start):
   row.idx = idx
```